src/formatting.rs

```diff
@@ -5,6 +5,7 @@ use std::io::{self, Write};
 use std::time::{Duration, Instant};
 
 use rustc_ast::ast;
+use rustc_ast::AstLike;
```

Comment:
I gather this is indeed needed otherwise the deny warnings compiler lint would've caused build failures, though curious what we need this for?

Reply:
Yeah we need the trait ``rustc_ast::AstLike`` to be in scope to write ``module.attrs()``. Here's the error message you get if you remove the ``use statement`` and run ``cargo check``

```
  --> src\formatting.rs:70:29
   |
70 |     if contains_skip(module.attrs()) {
   |                             ^^^^^ method not found in `&Module<'_>`
   |
   = help: items from traits can only be used if the trait is in scope
help: the following trait is implemented but not in scope; perhaps add a `use` for it:
   |
3  | use crate::rustc_ast::AstLike;
   |
```